### ISSUE TITLE
"Memory access fault by GPU node-1" error with RX 6600XT on Linux

### ISSUE THREAD

**Issue by JordyProvost** [easydiffusion/easydiffusion#1450]
Hello,

I have the exact same problem as the one described [here](https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/8139)

I'm on Linux Debian 12 wih an AMD RX6600XT.

I launch Easy SD with : 

```
export HSA_OVERRIDE_GFX_VERSION=10.3.0
./start.sh
```

When generating image, i have the following error :

```
home/username/easy-diffusion/installer_files/env/lib/python3.8/site-packages/torchsde/_brownian/brownian_interval.py:594: UserWarning: Should have tb<=t1 but got tb=14.611637115478516 and t1=14.611637.               | 0/25 [00:00<?, ?it/s]
  warnings.warn(f"Should have {tb_name}<=t1 but got {tb_name}={tb} and t1={self._end}.")
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 25/25 [03:06<00:00,  7.47s/it]
Memory access fault by GPU node-1 (Agent handle: 0x561631862c10) on address 0x7f5e00600000. Reason: Page not present or supervisor privilege.█████████████████████████████████████████████████████████████████████████████████| 25/25 [03:06<00:00,  6.47s/it]
/home/username/easy-diffusion/scripts/on_sd_start.sh : ligne 82 : 1549481 Abandon                 uvicorn main:server_api --app-dir "$SD_UI_PATH" --port "$ED_BIND_PORT" --host "$ED_BIND_IP" --log-level error
Press any key to continue
```

The Issue in the link above contain a patch for the stablediffusion webui.sh script, but i dont known how to apply it since this file does not exist on Easy StableDiffucion (i have not find it).

Thanks in advance.

Regards,

**Comment by JordyProvost** [comment]
Anyone to help me ? Thanks.

**Comment by JeLuF** [comment]
We've built a fix for this, implemented in PR #1491. The startup scripts now automatically detect the right settings based on the installed GPU. To update to the new version, run `./start.sh` (without any env settings).

**Comment by JordyProvost** [comment]
Oh, thank you, i can confirm it is working flawlessly with my GPU now ! :+1:
